File: Entity Framework/Kod/03/Product.cs
namespace ŁukaszDydekEFProducts
{
    internal class Product
    {
        public int ProductID { get; set; }
        public string ProductName { get; set; }
        public int UnitsOnStock { get; set; }

        public Product(string productName, int unitsOnStock) 
        {
            this.ProductName = productName;
            this.UnitsOnStock = unitsOnStock;
        }
    }
}

File: Hibernate & JPA/Kod/03/Z tabelą łącznikową/Product.java
package org.example;

import javax.persistence.*;

@Entity
public class Product {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private int productID;
    private String productName;
    private int unitsOnStock;

    public Product() {}

    public Product(String productName, int unitsOnStock) {
        this.productName = productName;
        this.unitsOnStock = unitsOnStock;
    }
}

